assets/javascripts/kitten/components/accordeon/__snapshots__/test.js.snap

```diff
@@ -1,128 +1,503 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`<Accordeon /> with one item matches with snapshot 1`] = `
-.c1 {
-  box-sizing: border-box;
+exports[`<Accordeon /> with an ID matches with snapshot 1`] = `
+Array [
+  .c1 {
+  display: block;
+  margin: 0;
+  width: 100%;
   padding: 1.25rem;
   min-height: 1.875rem;
+  box-sizing: border-box;
+  overflow: visible;
+  background: transparent;
   border: 0.125rem solid #eee;
   border-radius: 0.1875rem;
+  color: inherit;
+  font: inherit;
+  text-align: inherit;
+  line-height: normal;
+  -webkit-appearance: none;
   cursor: pointer;
+  outline: none;
+}
+
+.c1[aria-expanded='true'] {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.c3 {
+  margin-top: 1.875rem;
+  margin-bottom: 1.875rem;
+}
+
+.c0 {
+  margin-bottom: 0.9375rem;
 }
 
 .c2 {
-  display: none;
   box-sizing: border-box;
-  padding: 1.875rem;
+  visibility: visible;
+  overflow: hidden;
+  height: auto;
+  max-height: NaNrem;
```

Comment:
Etrange cette valeur.

Reply:
🤔  je pense qu'il n'a pas accès à la propriété `this.div.scrollHeight` du composant dans le test, vu qu'il ne joue qu'avec le code… à part s'il le passe dans un navigateur (en headless) pour le test, je ne vois pas comment il pourrait avoir accès à ce genre de propriété du DOM…

Reply:
Ca doit être dans le cas de la valeur par défaut. Il essaye de faire un `pxToRem('none')`.

Reply:
C'est modifié